test/unstable/packageModules.chpl

```diff
@@ -12,6 +12,7 @@ import CopyAggregation;
 import Crypto;
 import Curl;
 import DistributedBag;
+import DistributedBagDeprecated;
```

Comment:
This should be a test in `test/deprecated/` somewhere rather than in `test/unstable` (because DistributedBagDeprecated is deprecated rather than unstable). I would expect you can simply add a test that includes `import DistributedBagDeprecated;` and put the warning in the `.good` file.

Reply:
This test checks the `unstable` warnings of the package modules, so I added this line in order to reflect the new `DistributedBagDeprecated` package module. For some reasons, the deprecation warning is shown in `.good` instead of the unstable one, like for the `BinaryIO` module. 

Nevertheless, I agree that a standalone deprecation test is needed.

Reply:
The deprecation test has already been added in a previous commit.

Reply:
IMO it makes sense to remove `DistributedBagDeprecated` from this test altogether.